app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryListScreen.kt

```diff
@@ -200,17 +201,22 @@ fun GalleryListScreen(lub: ListUrlBuilder, navigator: DestinationsNavigator) {
     var searchCoverOnly by rememberSaveable { mutableStateOf(false) }
     var imagePath by rememberSaveable { mutableStateOf("") }
 
+    var updateSearchField by remember { mutableStateOf(true) }
```

Comment:
Nit:This doesn't need to be a SnapshotState

Reply:
> It will be reset to true on recomposition?

I mean, the change of this value will not trigger any recompose, so no need to operate on snapshot when r/w it, but it still need to be remembered.

Reply:
Maybe we can introduce another set of "mutableBooleanOf" no state
to remove this overhead

Reply:
Anyway this is a nitpick, we can do it later